docs/content/deploying-to-hippo.md

```diff
@@ -40,24 +41,13 @@ $ git checkout 64cf9334528f1975d7cbff207997d83cee4f19c2
 $ ./run_servers.sh
 ```
 
-Clone the Hippo repo locally and checkout the v0.6.2 tag. Set the `BINDLE_URL` environment variable from the `run_servers.sh` script. For now, we'll need to manuall update a few lines in the Nomad job in the project and then, run the `dotnet build` command to build the project. Change into the `src/Web` directory and use `dotnet run` to run the project.
+Clone the Hippo repo locally. The minimum version required for the Hippo server is v0.10.0. Set the `BINDLE_URL` environment variable from the `run_servers.sh` script. For now, we'll need to manuall update a few lines in the Nomad job in the project and then, run the `dotnet build` command to build the project. Change into the `src/Web` directory and use `dotnet run` to run the project.
```

Comment:
Should the second sentence be reworded now that users no longer need to manually update Hippo code?

Reply:
good catch! will update.